deploy/images/applications-rp/Dockerfile

```diff
@@ -1,22 +1,25 @@
-# Note: Using Alpine image to enable Git installation for Terraform module downloads.
+# Use Alpine image to enable Git installation for Terraform module downloads.
 # Switch to distroless when Terraform execution is moved to a separate container.
-FROM --platform=${TARGETPLATFORM:-linux/amd64} alpine:3.18
```

Comment:
Is this logic built in?

Reply:
In CI/CD I saw this warning with this line: `RedundantTargetPlatform: Setting platform to predefined ${TARGETPLATFORM:-linux/amd64} in FROM is redundant as this is the default behavior (line 2)`. And I decided to remove the `platform`.

Also: https://docs.docker.com/reference/build-checks/redundant-target-platform/.

Reply:
Very cool 👍